src/compiler/checker.ts

```diff
@@ -80,6 +80,9 @@ namespace ts {
         /** This will be set during calls to `getResolvedSignature` where services determines an apparent number of arguments greater than what is actually provided. */
         let apparentArgumentCount: number | undefined;
 
+        /** Whether user has not yet been warned about using BigInt without --experimentalBigInt */
+        let notWarnedAboutBigInt = true;

```

Comment:
OK, so after talking this one out with other ppl on the team, I believe this was a stupid suggestion on my side. 

IDEs ask for errors on files in no specific order. with this in place the error will keep appearing and disappearing randomly. i think we are better off having the error on every use. We already report errors on every use for decorators. so this should just be the same. I think we can just remove this check, and keep the error.

Sorry for the bad suggestion.